src/components/Footer/index.js

```diff
@@ -37,62 +39,84 @@ export const OfficeStyled = styled(Row)`
   overflow: hidden;
 `
 
-const Footer = () => {
-  return (
-    <GreyFooter>
-      <Padding top={{ smallPhone: 3, tablet: 4 }}>
+const GDPR = styled.div`
+  display: flex;
+  height: 48px;
+  align-items: center;
+  color: white;
+
+  a {
+    padding-right: 36px;
+  }
+`
+
+const LinkUnderline = styled(Link)`
+  text-decoration: underline;
+`
+
+const gdprLinksElement = gdpr.map(link => (
```

Comment:
Now that this isn't repeated anywhere, we can probably move this code directly into line 113 so that all the code is together

But it still might be nice to separate the Office section & FooterBanner sections into different files, just so that this file isn't so long?

Reply:
Absolutely! 👍 Done